weave/tests/trace/test_client_trace.py

```diff
@@ -105,7 +116,7 @@ def test_trace_server_call_start_and_end(client):
     start = tsi.StartedCallSchemaForInsert(
         project_id=client._project_id(),
         id=call_id,
-        op_name="test_name",
+        op_name="weave:///test_entity/test_project/op/test_name:test",
```

Comment:
question: it should very much be allowed for op_name to NOT be a ref - does that still work?

Reply:
yep, removing this change to make that explicit.